docs/book/how-to/create-and-run-templates/run-a-template.md

```diff
@@ -1,30 +1,51 @@
 ---
-description: Run a template
+description: From a template to a pipeline run
 ---
 
 # Run a template
 
-{% hint style="warning" %}
-This is a [ZenML Pro](https://zenml.io/pro) only feature. Please 
-[sign up here](https://cloud.zenml.io) get access.
+{% hint style="success" %}
+This is a [ZenML Pro](https://zenml.io/pro)-only feature. Please
+[sign up here](https://cloud.zenml.io) to get access.
 {% endhint %}
 
-<!-- ## Run a template from the dashboard -->
+Once your template is set up, you can **run** it and create a new pipeline run 
+that will be conducted on the same stack. Before it starts, you have a window
+to seamlessly readjust the configuration of your pipelines and steps.
 
-## Run a template in code
+## Run a template using the Python SDK
+
+You can use the ZenML client to run a template:
 
 ```python
 from zenml.client import Client
 
-template = Client().get_run_template(<NAME_OR_ID>)
+template = Client().get_run_template("TEMPLATE_NAME")
+
 config = template.config_template
-# Optionally modify the config here
+
+# [OPTIONAL] ---- modify the config here ----
 
 Client().trigger_pipeline(
     template_id=template.id,
     run_configuration=config,
 )
 ```
 
+## Run a template using the dashboard
+
+In order to run a template from the dashboard:
+
+- You can either click `Run a Pipeline` on the main `Pipelines` page, or
+- You can go to a specific template page and click on `Run Template`.
+
+Either way, you will be forwarded to a page where you will see the 
+`Run Details`. Here, you have the option to upload a `.yaml` [configurations
+file](https://docs.zenml.io/how-to/use-configuration-files) or change the 
+configuration on the go by using our editor **(powered with auto-completion and 
```

Comment:
The auto completion is not really working for anyone writing normal yaml right now

Reply:
Not sure if I get it correctly what you mean by `normal yaml`. How can I rephrase this to fit the current state of the implementation?